src/polygon.rs

```diff
@@ -614,19 +619,14 @@ where
 /// The result of one plane being cut by another one.
 /// The "cut" here is an attempt to classify a plane as being
 /// in front or in the back of another one.
-#[derive(Debug)]
-pub enum PlaneCut<T> {
+#[derive(Debug, PartialEq)]
+pub enum PlaneCut {
     /// The planes are one the same geometrical plane.
-    Sibling(T),
+    Sibling,
     /// Planes are different, thus we can either determine that
     /// our plane is completely in front/back of another one,
     /// or split it into these sub-groups.
```

Comment:
nitpick: It might be a bit confusing to keep referring to "planes" here when we're only working directly with `Polygon`. Not sure if there's an easy fix, though.